app/views/groups/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= render "partials/nav" %>
+<div class="button-flex gift">
+  <h1 class="coupons">Gift Coupons</h1>
+  <h3 class="coupons">$<%=Group.total(@group.id)%>.00</h3>
+</div>
+
+<%@relations.each do |relation|%>
+  <%entity = Entity.find(relation.entity_id)%>
+  <div class = "body-flex">
+  <img src=<%=@group.icon%> alt="icon">
+
+  <h3><%=entity.name%></h3>
+  <p><%=entity.created_at%></p>
+
+  <h3>$<%=entity.amount%>.00</h3>
+  <%= button_to "Delete", entity_path(entity.id), html: {'data-turbo' => "false"}, :method => :delete %>
+  </div>
+<%end%>
+
+<div class="button-flex">
+<%= link_to new_entity_path, :remote => true do %>
+  <input type="button" value="Add transaction">
+<% end %>
+<br>
```

Comment:
- [ ] Please make sure that the image is not taking much of the space and follow the design.

### Your design

![Screenshot from 2022-03-25 03-46-53](https://user-images.githubusercontent.com/22434660/160019963-c91c5970-c9e1-443c-aedb-52e7c1372990.png)

### Given design

![Screenshot from 2022-03-25 03-53-08](https://user-images.githubusercontent.com/22434660/160020014-ffc67554-144b-4dbf-ba97-557b0aa927f8.png)